src/components/panel/panel.js

```diff
@@ -710,7 +710,7 @@ MdPanelService.prototype.open = function(opt_config) {
  * @returns {MdPanelPosition}
  */
 MdPanelService.prototype.newPanelPosition = function() {
-  return new MdPanelPosition(this._$window);
+  return new MdPanelPosition(this._$window, this._$injector);
```

Comment:
If you're going to pass $injector, you can remove $window and just get that from $injector as well.

In general, if there were 3+ items, we used $injector, if there were less, we injected them directly.